src/configuration/Configuration.cpp

```diff
@@ -21,15 +21,7 @@
     THE SOFTWARE.
 */
 
-#ifdef ESP32
-    // #define CONFIG_LITTLEFS_SPIFFS_COMPAT 1
-    // #define CONFIG_LITTLEFS_LFS_YES_TRACE
-    // #define LFS_YES_TRACE
-    #include <LITTLEFS.h>
-    #define LittleFS LITTLEFS
-#else
-    #include <LittleFS.h>
-#endif
+#include <LittleFS.h>
```

Comment:
This isn't gonna work with the normal ESP32 builds

Reply:
how come?

Reply:
because you're not bumping the framework version for the "old" ESP32, in which this file doesn't exist. Which is why we had to use a library to provide this, which needs some defines to work properly.

Reply:
Is it resolved with the latest commits?